.github/workflows/KeyExists_release.yml

```
@@ -0,0 +1,13 @@
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: FrendsPlatform/FrendsTasks/.github/workflows/release.yml@main
+    with:
+      workdir: Frends.Redis.KeyExists
+    secrets:
+      feed_api_key: ${{ secrets.TASKS_FEED_API_KEY }}
+      
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)